foods/views.py

```diff
@@ -264,6 +269,7 @@ def cook_create(request):
     if request.method == "POST":
         name = request.POST.get("title")
         des = request.POST.get("description")
```

Comment:
For variable name, I for sure our member will understand the intention of variable.
But I would like to convince everyone to write descriptive name for variable.
Please change the variable name to be more specific for good practice.
Cheers!

Reply:
included 
`des`
`Ing`
`totalcal` I suggest to be something like `total_cal` or `total_calories`
`fatcal` suggested as above
`sugargrams` also as above